dds_web/web/user.py

```diff
@@ -354,9 +354,10 @@ def confirm_self_deletion(token):
     """Confirm user deletion."""
     s = itsdangerous.URLSafeTimedSerializer(flask.current_app.config.get("SECRET_KEY"))
 
+    # Get email from token
+    email = s.loads(token, salt="email-delete", max_age=604800)
+
```

Comment:
This should be in the try-except block, this is what generates the `itsdangerous.exc.SignatureExpired` exception etc that are caught in the except statements below.

Reply:
Fixed.